### PR TITLE
added dob

### PR DESCRIPTION
**Pull request by vamsi-krishna-m**

### CODE REVIEW

**Comment by vamsikrishna2636** [comment]
add some more description

**Comment by vamsi-krishna-m** [comment]
is all ok

**Comment by vamsikrishna2636** [comment]
everything perfect

**Review by vamsi-krishna-m — COMMENTED** [review]
add points

**Review by vamsikrishna2636 — COMMENTED** [review]
okkkkkkkkkkk

**Review by vamsi-krishna-m — COMMENTED** [review]
ok not bad

**Review by vamsikrishna2636 — CHANGES_REQUESTED** [review]
ghfhgfhgh

**Comment by vamsikrishna2636** [comment]
ksjdkgjksd

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # test
-practicing pull requests
+practicing pull requests.
+date of bith is added to vamsi file.
```
Comment (vamsikrishna2636) [inline README.md:3]: jai balayya

**Review by vamsikrishna2636 — COMMENTED** [review]
check the following comments

File: vamsi [hunk 2]
```diff
@@ -1 +1,3 @@
+kflklfdkbm
 mannam vamsi krishna
```
Comment (vamsikrishna2636) [inline vamsi:2]: hakil

**Pull request merged**